makefile

```diff
@@ -0,0 +1,8 @@
+run: build
+	./primary $(ARGS)
+build: 
+	gcc primary.c -o primary
```

Comment:
Build has no deps.
I think it means that `make build ` recompiles everything no matter if the files changed or not.